runtime/jcl/common/stdinit.c

```diff
@@ -161,6 +162,9 @@ restoreRunStandardInit(J9JavaVM *vm)
 	}
 	((J9NativeLibrary*)nativeLibrary)->flags |= J9NATIVELIB_FLAG_ALLOW_INL;
 
+	/* Run the JCL initialization code (what used to be JNI_OnLoad) */
+	nativeLibrary->send_lifecycle_event(vmThread, nativeLibrary, "JCL_OnLoad", JNI_VERSION_1_1);
```

Comment:
The other caller of this API looks like:
```
/* Run the JCL initialization code (what used to be JNI_OnLoad) */
	jniVersion = nativeLibrary->send_lifecycle_event(vmThread, nativeLibrary, "JCL_OnLoad", JNI_VERSION_1_1);
	if (!vmFuncs->jniVersionIsValid(jniVersion)) {
		return 1;
	}
```
Should this code validate the JNI version as well?

Reply:
yes, I've added the JNI version validation